spec/features/guest_spec.rb

```diff
@@ -60,7 +59,7 @@
     @user = User.last
     expect(@user.login).to eq('martha') 
     expect(@guest.id).to eq(@user.id)
-    expect(page.current_path). to eq collection_translate_page_path(@collection.owner, @collection, @work, @page.id)
+    expect(page.current_path). to eq collection_transcribe_page_path(@collection.owner, @collection, @work, @page.id)
```

Comment:
@tbl73 Can you review the `guest_spec` for sanity?  I believe that we should be sending guest users back to the transcribe page once they've registered, provided they were already on the transcribe page.  I'm not entirely sure how it worked at all before -- maybe one of my previous edits in this branch messed things up?

Reply:
I just noticed this - I'll take a look at it.  That sounds right, though.  I know generally we send users back to wherever they were before they register/log in, but it was slightly trickier with the guest users and making sure the page actually saved.